README.md

```diff
@@ -0,0 +1,6 @@
+Short instruction how to run weltyt.py in Docker:
```

Comment:
You can add `#` on the beginning of this line and it will be formatted while previewing the readme.
See this cheat sheet for more info about Markdown format: https://www.markdownguide.org/cheat-sheet/